source/_integrations/mqtt.markdown

```diff
@@ -49,9 +49,37 @@ This includes configuring SSL [certificate](/docs/mqtt/certificate/) options.
 - [Birth and last will messages](/docs/mqtt/birth_will/)
 - [Testing your setup](/docs/mqtt/testing/)
 - [Logging](/docs/mqtt/logging/)
+- [Using templates](/docs/configuration/templating/#using-templates-with-the-mqtt-integration)
 
 ## Event `event_mqtt_reloaded`
 
 Event `event_mqtt_reloaded` is fired when Manually configured MQTT entities have been reloaded and entities thus might have changed.
 
 This event has no additional data.
+
+## Supported platforms
+
+MQTT supports discovery and in in most cases a manual setup through the `configuration.yaml` file. The following platforms are supported.
+
+Platform | Discovery | Manual setup
+--- | --- | ---
+[Alarm control panel](/integrations/alarm_control_panel.mqtt/) | ✔ | ✔
```

Comment:
We really need to get rid of these MQTT platform pages soon (it is one of the few integrations that still has these).

Reply:
I'll see how we clean up this one.

Reply:
Do you pages like `/integrations/alarm_control_panel.mqtt/`?
What do other integrations do with platform documentation?

Reply:
In the past, almost all integrations have had their platform pages merged into a single page.  However, MQTT was one of the exceptions because it would be such a long page.

But the wish is still there to get rid of all those separate platform pages.

Reply:
@emontnemery What do you think about this. I think as a main page we should use the integration page (source/_integrations/mqtt.markdown). I would not advice too embed all platform pages yet. But we could migrate the `discovery.markdown` content to the integration page as well.